source/common/grpc/grpc_web_filter.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+#define SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+
+#include "envoy/http/filter.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/grpc/codec.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class GrpcWebFilter : public Http::StreamFilter {
+public:
+  GrpcWebFilter();
+  virtual ~GrpcWebFilter();
+
+  // GrpcWebFilter is neither copyable nor movable.
```

Comment:
You can derive from https://github.com/lyft/envoy/blob/master/source/common/common/non_copyable.h